pkg/osbuild/systemd_unit_create_stage.go

```diff
@@ -177,3 +229,112 @@ func NewSystemdUnitCreateStage(options *SystemdUnitCreateStageOptions) *Stage {
 		Options: options,
 	}
 }
+
+// GenSystemdMountStages generates a collection of
+// org.osbuild.systemd.unit.create stages with options to create systemd mount
+// units, one for each mountpoint in the partition table.
+func GenSystemdMountStages(pt *disk.PartitionTable) ([]*Stage, error) {
+	mountStages := make([]*Stage, 0)
+	unitNames := make([]string, 0)
+
+	genOption := func(ent disk.FSTabEntity, path []disk.Entity) error {
+		fsSpec := ent.GetFSSpec()
+		fsOptions, err := ent.GetFSTabOptions()
+		if err != nil {
+			return err
+		}
+
+		var filename string
+		var options *SystemdUnitCreateStageOptions
+		device := fmt.Sprintf("/dev/disk/by-uuid/%s", strings.ToLower(fsSpec.UUID))
+		if err := uuid.Validate(fsSpec.UUID); err != nil {
```

Comment:
This feels weird, are all potential errors from `uuid.Validate` still valid `by-uuid` identifiers?

Reply:
You're right, it is a bit weird.  It's difficult to tell, really.  I don't know what the rules are for the `by-uuid` links.  I think I'll add regex check for the FAT volid format instead.

Reply:
Done.